File: api-gateway/api-gateway/Model/Info/InfoModel.cs
using System.ComponentModel.DataAnnotations;

namespace api_gateway.Model.Info
{
    public class InfoModel
    {
        public Guid id { get; set; }

        public string name { get; set; }

        public string cif { get; set; }

        public string deliveryroom { get; set; }

        public string affarisofficer { get; set; }

        public string recive { get; set; }

        public int status { get; set; }

        public string note { get; set; }

        public string date { get; set; }


    }
}

File: back-end/src/models/Info/Info.model.js
const mongoose = require("mongoose")

const Schema = mongoose.Schema

const InfoSchema = new Schema({
    id: { type: String, require: false },
    cif: { type: Number, require: true },
    time: { type: Date },
    deliveryroom: { type: String, require: false },
    affairsofficer: { type: String, require: false },
    recive: { type: String, require: false },
    status: { type: String, require: false },
    note: { type: String, require: false },
    name: { type: String, require: false },
}, { timestamps: true });

const InfoModel = mongoose.model("curriculum", InfoSchema);
module.exports = InfoModel 